code/code/sys/migrations.cc

```diff
@@ -61,6 +60,30 @@ void runMigrations() {
                 }
             }
         },
+        [&](){
+            //support tweaks db
+            vlogf(LOG_MISC, "Adding tweak table to DB");
+            sneezy.query(
+                    "create table globaltweaks ("
+                    "tweak_id int primary key auto_increment not null, "
+                    "tweak_type int not null, "
+                    "tweak_value float(20) not null, "
+                    "tweak_target float(20) not null, "
+                    "tweak_rate float(20) not null, "
+                    "datecreated datetime not null default CURRENT_TIMESTAMP)"
+                    );
+
+            //this doesn't work / migrations run before stats load      
+            //sneezy.query("insert into globaltweaks (tweak_type, tweak_value, tweak_target, tweak_rate) values ('1','%f','%f','%f')", stats.equip, stats.global_lp_target, stats.global_lp_target_changerate);
+            //another way would be to parse the stats file for current stats.equip value etc., but maybe an admin can just set the value after migration...
```

Comment:
The right way would be to load the stats file, if not unreasonably complex. If too complex, hardcode the defaults in constructor with a clean but migrated DB.

Reply:
Stats.equip is buried in the middle of txt/stats so it would essentially be almost a copy of the whole file load.  Also I should probably purge stats.equip out of the file load and save while here, but I would like to put that off until I have moved more things out of stats.

Reply:
Isn't it possible to call some `readStatsFile()` or such?